src/main/java/com/puppycrawl/tools/checkstyle/api/AutomaticBean.java

```diff
@@ -291,7 +291,7 @@ protected void setupChild(Configuration childConf)
     /** A converter that converts strings to patterns. */
     private static class PatternConverter implements Converter {
 
-        @SuppressWarnings({"unchecked", "rawtypes"})
```

Comment:
Redundant suppression

Reply:
Why is this redundant when we have `Class` with no parameter?